traceapp/tmpl/doc.go

```diff
@@ -0,0 +1,6 @@
+// Package tmpl contains template data used by appdash.
```

Comment:
This should be Appdash, not appdash. Similar to https://github.com/shurcooL/appdash-data/commit/efa2841cf2704832e92b58a87c79bf832f7f2115.